packages/remark-mdxs/index.js

```diff
@@ -0,0 +1,76 @@
+const visit = require('unist-util-visit')
+const remove = require('unist-util-remove')
+const {toJSX} = require('@mdx-js/mdx/mdx-hast-to-jsx')
+
+module.exports = function({delimiter = 'hr'}) {
```

Comment:
Just curious, I'm using `thematicBreak` in `@mdx-deck/mdx-plugin` but with the way the delimiter is used below will either `hr` or `thematicBreak` work?

Reply:
As is it won't work because we're operating on MDXHAST (since it's acting as a custom compiler for MDX). But it might make sense to make MDXS more standalone and not use `mdx` core and `remark-mdx` directly instead. That would allow us to operate on the MDXAST.

Reply:
Regarding these architectural things, if `remark-mdxs` (or a different implementation of MDXs) were to be implemented today, do either of you (or @ChristianMurphy or @wooorm) have any recommendations for trying to make a new MDXs PR for MDX v2?

@EricCote or @ProchaLu (or both) may take a shot at implementing something like this in the next weeks, see this:

- https://github.com/0phoff/MDXP/issues/12#issuecomment-1359354136